samples/highcharts/demo/music-race-chart/demo.js

```diff
@@ -0,0 +1,196 @@
+const btn = document.getElementById('play-pause-button'),
+    input = document.getElementById('play-range'),
+    startYear = 1973,
+    endYear = 2021;
+
+// General helper functions
+const getRevenue = (record, year) => {
+    const revenue = record[year];
+    if (typeof revenue !== 'undefined') {
+        return revenue;
+    }
+    return 0;
+};
+
+const arrToAssociative = arr => {
+    const tmp = {};
+    arr.forEach(item => {
+        if (item[1] !== null) {
+            tmp[item[0]] = item[1];
+        }
+    });
+
+    return tmp;
+};
+
+const formatRevenue = [];
+
+const chart = Highcharts.chart('container', {
+    chart: {
+        type: 'area'
+    },
+    title: {
+        text: 'Music revenue race chart'
+    },
+    data: {
+        csv: document.getElementById('csv').innerHTML,
+        itemDelimiter: '\t',
+        complete: function (options) {
+            for (let i = 0; i < options.series.length; i++) {
+                formatRevenue[i] = arrToAssociative(options.series[i].data);
+                options.series[i].data = null;
+            }
+        }
+    },
+    xAxis: {
+        allowDecimals: false,
+        min: startYear,
+        max: endYear
+    },
+    yAxis: {
+        title: {
+            text: 'Revenue in the U.S.'
+        },
+        labels: {
+            format: '${text} B'
+        }
```

Comment:
On the mobile, it would buy us some plotting space if we reorganized this. Add a responsive rule, then put the labels on top of the grid lines. The title could probably be placed on top of the axis.